### ISSUE TITLE
 have a lot of error while archiving my app 

### ISSUE THREAD

**Issue by mohamedhosni250** [inspireui/support#4353]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues: 
+ Product & version:
+ Flutter (or React Native) version: 
+ Testing Device/Simulator: 
+ Screenshot issues (drag the file to attach here): 

i have a lot of error while archiving my app in Xcode =with web view package and  bitcode any help!

https://ibb.co/9Y67pVQ check this screenshot plz 

i have latest flutter version & fluxstore 

**_Step 2 (require): submit proof of purchasing the license on http://verify.inspireui.com_**

**_Important Note:_**
- Please help to ready the support policies before creating a new ticket: https://inspireui.com/support-policies/
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by ameenbadri7** [comment]
Thank you for the feedback.

Please help us to submit this form and fill all required info to continue with the support - http://verify.inspireui.com/ - please don't share the purchase code or screenshot public here as another user will use it for activation.

We will close this ticket in 24 hours if it is not provided enough detail, kindly reopen the ticket after you have done the form submission. 

Thank you for your time!

**Comment by mohamedhosni250** [comment]
I have been submitted

**Comment by minhcasi** [comment]
@mohamedhosni250 

Please make sure to run `flutter build ios` before select Archive from XCode

**Comment by mohamedhosni250** [comment]
error:
    /lib/flutter-stable/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.2
    2+1/ios/Classes/JavaScriptChannelHandler.h: No such file or directory (in
    target 'webview_flutter' from project 'Pods')
    error: the following command failed with exit code 1 but produced no further
    output
    CpHeader
    /lib/flutter-stable/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.2
    2+1/ios/Classes/JavaScriptChannelHandler.h
    /Users/mohamedhosni/Desktop/petshouses/build/ios/Release-iphoneos/webview_fl
    utter/webview_flutter.framework/Headers/JavaScriptChannelHandler.h
    error:
    /lib/flutter-stable/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.2
    2+1/ios/Classes/FlutterWebView.h: No such file or directory (in target
    'webview_flutter' from project 'Pods')
    error: the following command failed with exit code 1 but produced no further
    output
    CpHeader
    /lib/flutter-stable/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.2
    2+1/ios/Classes/FlutterWebView.h
    /Users/mohamedhosni/Desktop/petshouses/build/ios/Release-iphoneos/webview_fl
    utter/webview_flutter.framework/Headers/FlutterWebView.h
    error:
    /lib/flutter-stable/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.2
    2+1/ios/Classes/FLTWebViewFlutterPlugin.h: No such file or directory (in
    target 'webview_flutter' from project 'Pods')
    error: the following command failed with exit code 1 but produced no further
    output
    CpHeader
    /lib/flutter-stable/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.2
    2+1/ios/Classes/FLTWebViewFlutterPlugin.h
    /Users/mohamedhosni/Desktop/petshouses/build/ios/Release-iphoneos/webview_fl
    utter/webview_flutter.framework/Headers/FLTWebViewFlutterPlugin.h
    error:
    /lib/flutter-stable/.pub-cache/hosted/pub.dartlang.org/webview_flutter-0.3.2
    2+1/ios/Classes/FLTWKNavigationDelegate.h: No such file or directory (in
    target 'webview_flutter' from project 'Pods')
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
 error output white do flutter build ios

**Comment by minhcasi** [comment]
@mohamedhosni250 - could I check if you have run `flutter clean` and update the latest `pod`? please double check this guide https://gist.github.com/minhcasi/2362b8ed369738cea2bf10a57ac569e1

**Comment by minhcasi** [comment]
We have checked the archive and everything works fine

<img width="1303" alt="Screen Shot 2020-07-27 at 11 06 33" src="https://user-images.githubusercontent.com/1459805/88502674-6403d300-cff9-11ea-8342-c5e66a9dd778.png">

**Comment by minhcasi** [comment]
@mohamedhosni250 - let us know if you have any questions - thank you!
